sci-astronomy/predict/predict-2.2.3-r2.ebuild

```diff
@@ -0,0 +1,207 @@
+# Copyright 1999-2016 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=6
+
+inherit toolchain-funcs autotools
+
+DEB_P=${PN}_${PV}
+DEB_PR=3.1
+
+DESCRIPTION="Satellite tracking and orbital prediction"
+HOMEPAGE="http://www.qsl.net/kd2bd/predict.html"
+SRC_URI="mirror://debian/pool/main/${PN:0:1}/${PN}/${DEB_P}.orig.tar.gz
+	mirror://debian/pool/main/${PN:0:1}/${PN}/${DEB_P}-${DEB_PR}.diff.gz"
+
+LICENSE="GPL-2"
+SLOT="0"
+IUSE="doc gtk nls xforms xplanet"
+KEYWORDS="~amd64 ~ppc ~x86 ~amd64-linux ~x86-linux"
+
+RDEPEND="sys-libs/ncurses:0/5
+	gtk? ( x11-libs/gtk+:2 )
+	xforms? ( x11-libs/xforms )
+	xplanet? ( x11-misc/xplanet[truetype] )"
+DEPEND="${RDEPEND}"
+
+PATCHES=(
+	"${FILESDIR}"/"${P}"-earthtrack.patch
+	"${WORKDIR}"/${DEB_P}-${DEB_PR}.diff
+)
+
+src_prepare() {
+	default
+	sed -i -e 's:predict\(.*\)/:predict-2.2.3\1/:g' \
+		debian/patches/140*.diff || die
+	sed -i -e 's:\(a\|b\)/:predict-2.2.3/:g' \
+		debian/patches/180*.diff || die
+	local patch
+	for patch in $(cat debian/patches/series) ; do
+		eapply "debian/patches/${patch}"
+	done
+	# fix some further array out of bounds errors
+	sed -i -e "s/satname\[ 26/satname\[ 25/g" \
+		clients/gsat-1.1.0/src/db.c || die
+	sed -i -e "s/satname\[ 26/satname\[ 25/g" \
+		clients/gsat-1.1.0/src/comms.c || die
+	sed -i -e "s/output\[20\];/output[21];/" \
+		utils/moontracker/moontracker.c || die
+	# fix underlinking
+	sed -i -e '/AC_OUTPUT/i \
+AC_CHECK_LIB(m,cos) \
```

Comment:
Also, prefer `AC_SEARCH_LIBS` over `AC_CHECK_LIB`: https://autotools.io/autoconf/finding.html

Furthermore, arguments of Autoconf usually need to be "quoted", that is, in brackets:
```
AC_SEARCH_LIBS([m], [cos])
```